### PR TITLE
Add the ability to filter out farmers with no parcels

### PR DESCRIPTION
**Pull request by andreighh**
Refs: #360

### CODE REVIEW

File: tests/e2e/specs/UsersStaff.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    browser.page
+      .usersStaff()
+      .waitForElementVisible("@checkboxes")
+      .section.checkboxes.expect.element("@hideInactiveCheckbox").to.be
+      .selected;
+    browser.end();
+  },
+  "check inactive checkbox": (browser) => {
+    browser.load().staffMock().logIn();
+    const user_staff_page = browser.page.usersStaff();
```
Comment (alexkiro) [inline tests/e2e/specs/UsersStaff.test.js:40]: Let's use camelCase here.

File: tests/e2e/specs/UsersStaff.test.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      .waitForElementVisible("@checkboxes")
+      .section.checkboxes.click("@hideInactive");
+    user_staff_page
+      .waitForElementVisible("@userTable")
+      .section.userTable.assert.elementCount("@row", 3);
+    browser.end();
+  },
+  "check no parcels checkbox": (browser) => {
+    browser.load().staffMock().logIn();
+    const user_staff_page = browser.page.usersStaff();
```
Comment (alexkiro) [inline tests/e2e/specs/UsersStaff.test.js:51]: Same here.

**Review by alexkiro — CHANGES_REQUESTED** [review]
Amazing work! Tests looks wonderful! 

---

Only minor style issue, and we're good to deploy.

File: tests/e2e/specs/UsersStaff.test.js (hunk 1 above)
Comment (andreighh) [reply tests/e2e/specs/UsersStaff.test.js:40]: 🤦🏻‍♂️

**Review by alexkiro — APPROVED** [review]

**Pull request merged**